docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Install & Configure
+toc: true
+weight: 1
+indent: true
+---
+
+# Install & Configure Quickstart
+
+Crossplane can be easily installed into any existing Kubernetes cluster using
+the regularly published Helm chart. The Helm chart contains all the custom
+resources and controllers needed to deploy and configure Crossplane.
+
+See [Install] and [Configure] docs for installing alternate versions and more
+detailed instructions.
+
+## Get a Kubernetes Cluster
+
+<ul class="nav nav-tabs">
+<li class="active"><a href="#setup-mac-brew" data-toggle="tab">macOS via Homebrew</a></li>
+<li><a href="#setup-mac-linux" data-toggle="tab">macOS / Linux</a></li>
+<li><a href="#setup-mac-linux" data-toggle="tab">Windows</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="setup-mac-brew" markdown="1">
+```
+brew upgrade
+brew install kind
+brew install kubectl
+brew install helm
+
+kind create cluster --image kindest/node:v1.15.11 --wait 5m
+```
+</div>
+<div class="tab-pane fade" id="setup-mac-linux" markdown="1">
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+</div>
+<div class="tab-pane fade" id="setup-windows" markdown="1">
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+</div>
+</div>
+
+## Install Crossplane
+Install the alpha version of Crossplane, which is suitable for community use and testing:
+<ul class="nav nav-tabs">
+<li class="active"><a href="#install-tab-helm3" data-toggle="tab">Helm 3</a></li>
+<li><a href="#install-tab-helm2" data-toggle="tab">Helm 2</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="install-tab-helm3" markdown="1">
+```
+kubectl create namespace crossplane-system
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+</div>
+<div class="tab-pane fade" id="install-tab-helm2" markdown="1">
+```
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+</div>
+</div>
+
+## Install Crossplane CLI
+The Crossplane CLI adds a set of `kubectl crossplane` commands to simplify common tasks:
+```
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane-cli/master/bootstrap.sh | bash
```

Comment:
@prasek do you think we should do another release of `crossplane-cli` prior to these docs going out? Or maybe just using `master` is fine?

Reply:
@hasheddan IMO yes we should cut another crossplane-cli if that is how stable versions are released.

Reply:
SGTM :+1: